package.json

```diff
@@ -51,7 +53,6 @@
     "react-router-dom": "^5.1.2",
     "react-select": "^5.4.0",
     "react-simplemde-editor": "^4.1.0",
-    "react-toggle-button": "^2.2.0",
```

Comment:
Removing react-toggle-button as I don’t see its usage in any files and this package is 5 yrs old - not maintained

cc @seaerchin @kishore03109 @alexanderleegs